test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerTestFixture.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using Microsoft.AspNetCore.Razor.Tools;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public class BuildServerTestFixture : IDisposable
+    {
+        private static readonly TimeSpan _defaultShutdownTimeout = TimeSpan.FromSeconds(60);
+
+        public BuildServerTestFixture()
+        {
+            PipeName = Guid.NewGuid().ToString();
+        }
```

Comment:
To make sure I understand this fixture correctly. The fixture relies on a test booting the rzc server and then the fixture tears it down yes? 

If that's the case I'd suggest booting the server as part of the fixture to not have variance in the tests.

Reply:
> To make sure I understand this fixture correctly. The fixture relies on a test booting the rzc server and then the fixture tears it down yes?
Yes

I thought about that. But it seemed better to let the tests do it as it is a real scenario and testing it is also useful. There  won't be much variance between tests other than the time taken.

Reply:
Testing bootup of rzc is definitely valuable but I'd split that into its own test which doesn't use the fixture.

The problem with relying on a random test booting rzc is if there's issues in booting rzc you'll end up getting false failures for tests. 

Also, by having the boot logic in the fixture it makes the fixture more understandable for future readers.